internal/modules/overview/logviewer/logviewer.go

```diff
@@ -47,6 +47,10 @@ func ToComponent(object runtime.Object) (component.Component, error) {
 		containerNames = append(containerNames, c.Name)
 	}
 
+	for _, c := range pod.Spec.EphemeralContainers {
+		containerNames = append(containerNames, c.Name)
```

Comment:
It can be handled in a separate PR, but the debugger container should be shown in the summary view.